File: TextSearchNet/model/IndexStats.cs
using System;
using Newtonsoft.Json;

namespace com.hideakin.textsearch.model
{
    public class IndexStats
    {
        [JsonProperty("gid")]
        public int Gid { get; set; }

        [JsonProperty("group")]
        public String Group { get; set; }

        [JsonProperty("file_count")]
        public int FileCount { get; set; }

        [JsonProperty("text_count")]
        public int TextCount { get; set; }

        [JsonProperty("total_bytes")]
        public long TotalBytes { get; set; }

        [JsonProperty("total_count")]
        public long TotalCount { get; set; }

        public override string ToString()
        {
            if (TotalCount == 0)
            {
                return string.Format("gid={0} group={1} files={2:#,0} texts={3:#,0} bytes={4:#,0} data={5:#,0}",
                    Gid, Group, FileCount, TextCount, TotalBytes, TotalCount);
            }
            return string.Format("gid={0} group={1} files={2:#,0} texts={3:#,0} bytes={4:#,0} data={5:#,0} ({6}%)",
                Gid, Group, FileCount, TextCount, TotalBytes, TotalCount, 100 * TotalBytes / (4 * TotalCount));
        }
    }
}

File: TextIndexApi/src/main/java/com/hideakin/textsearch/index/model/IndexStats.java
package com.hideakin.textsearch.index.model;

import com.fasterxml.jackson.annotation.JsonProperty;

public class IndexStats {

	@JsonProperty("gid")
	private int gid;
	
	@JsonProperty("group")
	private String group;

	@JsonProperty("file_count")
	private int fileCount;

	@JsonProperty("text_count")
	private int textCount;

	@JsonProperty("total_bytes")
	private long totalBytes;

	@JsonProperty("total_count")
	private long totalCount;

	public int getGid() {
		return gid;
	}

	public void setGid(int gid) {
		this.gid = gid;
	}

	public String getGroup() {
		return group;
	}

	public void setGroup(String group) {
		this.group = group;
	}

	public int getFileCount() {
		return fileCount;
	}

	public void setFileCount(int fileCount) {
		this.fileCount = fileCount;
	}

	public int getTextCount() {
		return textCount;
	}

	public void setTextCount(int textCount) {
		this.textCount = textCount;
	}

	public long getTotalBytes() {
		return totalBytes;
	}

	public void setTotalBytes(long totalBytes) {
		this.totalBytes = totalBytes;
	}

	public long getTotalCount() {
		return totalCount;
	}

	public void setTotalCount(long totalCount) {
		this.totalCount = totalCount;
	}

}
